package.json

```diff
@@ -2,6 +2,7 @@
   "name": "easybloqs",
   "author": "Leaphy Robotics",
   "description": "Build Leaphy Arduino programs",
+  "productName": "Leaphy Easybloqs",
```

Comment:
Apparently productName is also used in file paths and this breaks the build because spaces are not properly escaped :-( 
Put in a hypen?

```
gyp info spawn args [ 'BUILDTYPE=Release', '-C', 'build' ]
make: Entering directory '/tmp/electron-packager/linux-x64/Leaphy Easybloqs-linux-x64/resources/app/node_modules/@serialport/bindings/build'
  CXX(target) Release/obj.target/bindings/src/serialport.o
g++: error: Easybloqs-linux-x64/resources/app/node_modules/@serialport/bindings/8.5.5/include/node: No such file or directory
```

Reply:
From the Trello card the requirement is pretty clear that the app Id should be "Leaphy Easybloqs". So I think we are stuck with overriding it in the packager config.